Tests/Unit/Content/ContentTypeResolver/SmartContentResolverTest.php

```diff
@@ -79,7 +80,12 @@ protected function setUp(): void
         $this->targetGroupStore = $this->prophesize(TargetGroupStoreInterface::class);
 
         $this->smartContentResolver = new SmartContentResolver(
-            ['media' => $this->mediaProviderResolver->reveal()],
+            new RewindableGenerator(
```

Comment:
you probably do not need to change this when using `iterable` as type of the argument above

Reply:
It is not really possible without